rasa/utils/tensorflow/layers.py

```diff
@@ -1256,7 +1256,7 @@ def __init__(
             model_confidence=model_confidence,
         )
 
-    def call(
+    def call(  # type: ignore[override]
```

Comment:
yeah these 2 are a bit tricky, I don't think there's another solution but to ignore